src/Pages/Authentication/Login/Login.jsx

```diff
@@ -0,0 +1,115 @@
+import React, { useState } from 'react';
+import { Link } from "react-router-dom";
+import '../index.css';
+import { Input, InputField } from '../../../Shared';
+import blitheCollage from '../../../Assets/Images/Blithe-collage.jpg';
+import blitheLogo from '../../../Assets/Images/Blithe-logo.png';
+import axios from 'axios';
+import { useNavigate } from "react-router-dom";
+import { actionLogin } from '../../../Helpers/Services/actions';
+import { v4 as uuid } from "uuid";
+import { useAlert } from '../../../Context/AlertContext';
+
+const Login = () =>{
+
+    const navigate = useNavigate();
+    const {alertContent , setAlertContent} = useAlert();
+    const [showPassword , setShowPassword ] = useState(false);
+    const [loginData, setLoginData] = useState(
+        {   
+            email:'',
+            password:''  
+        })
+
+    //Handle show/hide password
+    const handlePassword = () =>{
+        setShowPassword(!showPassword)
+    }
+
+
+    const handleLogin = async (e) =>{
+        e.preventDefault();
+
+        actionLogin(loginData)
+        .then((response) => {
+          console.log("resp........", response);
+          if (response.data && response.status === 200) {
+             navigate("/");
+             setAlertContent({_id: uuid(), isShow:true, type:'SUCCESS', content:"You have been loggen in successfully!"})
+          } else{
+            setAlertContent({_id: uuid(), isShow:true, type:'ERROR', content:"Unexpected error.Login Failed"})
+          }
+        })
+        .catch((response) => {
+          if (response.data && response.data.error) {
+            setAlertContent({_id: uuid(), isShow:true, type:'ERROR', content:"Unexpected error.Login Failed"})
+          }
+        })
+    }
```

Comment:
can use try catch, trying to understand why have you used .then .catch here

Reply:
ok! ty for the suggestion